source/conf.py

```diff
@@ -41,12 +40,24 @@
     "sphinx_copybutton",
     "sphinx.ext.graphviz",
     "sphinxcontrib.mermaid",
+    "sphinx.ext.intersphinx",
 ]
 
 # Extension Configuration
 graphviz_output_format = "svg"
 autosectionlabel_prefix_document = True
 
+# Intersphinx Extension Configuration
+rtd_language = os.environ.get("READTHEDOCS_LANGUAGE", "en")
+rtd_version = os.environ.get("READTHEDOCS_VERSION", "latest")
+
+intersphinx_mapping = {
+    "DoneXBlock": (
+        f"https://docs.openedx.org/projects/donexblock/{rtd_language}/{rtd_version}",
+        None,
+    ),
+}
```

Comment:
How big do we expect this list to get someday? Does it make sense to move this somewhere else now or later?

Reply:
If it ever gets too big for this, yea we can move it, but we can defer that until the problem exists.